boxwise_flask/graph_ql/resolvers.py

```diff
@@ -30,38 +32,40 @@ def serialize_date(value):
 # registers this fn as a resolver for the "allBases" field, can use it as the
 # resolver for more than one thing by just adding more decorators
 @query.field("allBases")
-def resolve_all_camps(_, info):
+def resolve_all_bases(_, info):
     # discard the first input because it belongs to a root type (Query, Mutation,
     # Subscription). Otherwise it would be a value returned by a parent resolver.
-    response = Bases.get_all_bases()
+    response = Base.get_all_bases()
     return list(response.dicts())
 
 
 # not everyone can see all the bases
 # see the comment in https://github.com/boxwise/boxwise-flask/pull/19
 @query.field("orgBases")
 def resolve_org_bases(_, info, org_id):
-    response = Bases.get_camps_by_org_id(org_id)
+    response = Base.get_for_organisation(org_id)
     return list(response.dicts())
 
 
 @query.field("base")
 def resolve_camp(_, info, id):
     authorization_test("bases", base_id=id)
-    response = Bases.get_camp(id)
+    response = Base.get_from_id(id)
```

Comment:
matter of prefernece again, but now it's less consistent with for instance, "User.get_user(email)"